eclass/tests/python-utils-bench.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+source tests-common.sh || exit
+
+export LC_ALL=C
+
+ITERATIONS=10000
+RUNS=3
+
+doit() {
+	local i
+	for (( i = 0; i < ITERATIONS; i++ )); do
+		"${@}"
+	done
+}
+
+timeit() {
```

Comment:
I might be missing something, but is the code here generic?
It looks to me like yes, so why not move it to something like test utils so other files can use it?

Reply:
It might be but not that eapi7-ver bench has different kind of output, and pypi has extra einfo explaining which case is being tested.